src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  --main-color1: #e1e7ed;
+  --main-color2: #3d4e61;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  justify-content: center;
+  margin: 20px 0;
+}
+
+.search_label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search_label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search_label_big:after {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search_label_small:after {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  height: 100%;
+  border: 1px solid var(--main-color1);
+  border-radius: 4px;
+  color: var(--main-color2);
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search::placeholder {
+  color: var(--main-color2);
+}
+
+.search:hover {
+  border: 1px solid var(--main-color1);
```

Comment:
```suggestion
  border-color: var(--main-color1);
```

Reply:
if remove these rules, the border will be different  
/*border-width: 1px;
  border-style: solid;*/